presto-i18n-functions/src/main/java/com/facebook/presto/i18n/functions/I18nMyanmarFunctions.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.i18n.functions;
+
+import com.facebook.presto.spi.function.Description;
+import com.facebook.presto.spi.function.ScalarFunction;
+import com.facebook.presto.spi.function.SqlType;
+import com.facebook.presto.spi.type.StandardTypes;
+import com.google.myanmartools.TransliterateZ2U;
+import com.google.myanmartools.ZawgyiDetector;
+import io.airlift.slice.Slice;
+
+import static io.airlift.slice.Slices.utf8Slice;
+
+public final class I18nMyanmarFunctions
+{
+    private I18nMyanmarFunctions() {}
+
+    static ZawgyiDetector detector = new ZawgyiDetector();
+    static TransliterateZ2U z2uTransliterator = new TransliterateZ2U("Zawgyi to Unicode");
+
+    /**
+     * ZawgyiDetector returns a confidence score that a given string is Zawgyi encoded.
+     * This constant defines the default confidence at which a string is treated as Zawgyi.
+     */
+    private static final double ZAWGYI_PROBABILITY_THRESHOLD = 0.9;
+
+    @Description("labels whether input strings use Unicode or Zawgyi font encoding")
+    @ScalarFunction
+    @SqlType(StandardTypes.VARCHAR)
+    public static Slice myanmarFontEncoding(@SqlType(StandardTypes.VARCHAR) Slice slice)
+    {
+        if (detector.getZawgyiProbability(slice.toStringUtf8()) > 0.9) {
```

Comment:
Do you want to use the constant `ZAWGYI_PROBABILITY_THRESHOLD ` here as well?

Reply:
ah yes! thanks for noticing will patch that